config/initializers/vault.rb

```diff
@@ -13,6 +22,21 @@ class VaultClient < Vault::Client
       read_timeout: 2
     }.freeze
 
+    def self.auth_token(vault_server)
+      uri = URI.parse(vault_server)
+      @http = Net::HTTP.start(uri.host, uri.port, DEFAULT_CLIENT_OPTIONS)
+      response = @http.request(Net::HTTP::Post.new(CERT_AUTH_PATH))
+      if response.code == "200"
+        JSON.parse(response.body).fetch("auth")["client_token"]
+      else
+        raise "Failed to get auth token from vault server"
```

Comment:
This isn't a new method. I just moved it up in the file, so the class methods appear before the initialize method.